charts/fybrik/templates/manager-deployment.yaml

```diff
@@ -96,6 +96,20 @@ spec:
                 name: razee-credentials
             {{- end }}
           env:
+            - name: TLS_ENABLED
+              value: {{ .Values.global.tls.enabled | quote | toString }}
+            {{- if .Values.global.tls.enabled }}
+            - name: MTLS_ENABLED
+              value: {{ .Values.global.tls.mtlsEnabled | quote | toString }}
+            - name: CERT_SECRET_NAME
+              value: {{ .Values.manager.tls.certs.clientCertSecretName }}
+            - name: CERT_SECRET_NAMESPACE
```

Comment:
do we really need separate secrets and namespaces for  certificates and CA?

Reply:
the certificate's secrets can be deployed in any namespace..  I think the data catalog and policy manager can also be deployed in any namespace in the coordinator cluster. we can use `<name>/<namespace>` but I think splitting them is clearer.